not-to-release/sources/answers/20111108084119AAhsBk8_ans.xml.conllu

```diff
@@ -7,7 +7,7 @@
 4	crashes	crash	VERB	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	0:root	_
 5	with	with	ADP	IN	_	8	case	8:case	_
 6	most	most	ADJ	JJS	Degree=Sup	8	amod	8:amod	_
-7	custom	custom	NOUN	NN	Number=Sing	8	compound	8:compound	_
```

Comment:
Not sure about _custom_ either. I think it also sounds really weird when it is conjoined with another `ADJ`. Or does something like _custom and updated ROM_ sound okay to you?

Reply:
I found attestations in COCA:

* Because direct observation of prices for custom and own-account software has not been possible
* he really has no business making and designing shoes -- not the custom and ready-to-wear lines he sells under his own name

It seems to me that this sense of 'custom' is almost always used prenominally (or in a participial compound like *custom-made*), so it probably started out as a noun-noun compound but semantically has grown distinct from the ordinary senses of 'custom'. I suspect it's on its way to being fully grammaticalized as an adjective. A couple of predicative ones in COCA:

* Each Bamboo is custom, with tubes selected according to rider weight.
* Everything is custom, from state-of-the-art lighting, ... to cabinetry using antique wood

A dictionary entry calling it an adjective: http://www.dictionary.com/browse/custom

Reply:
Ok, the COCA examples convinced me :)